django_netjsonconfig/base/device.py

```diff
@@ -48,6 +49,12 @@ class AbstractDevice(BaseModel):
                                                  null=True,
                                                  help_text=_('ip address of the management interface, '
                                                              'if available'))
+    additional_context = JSONField(null=True,
+                                   blank=True,
+                                   help_text=_('Additional \
+                                   <a href="http://netjsonconfig.openwisp.org \
+                                   en/stable/general/basics.html#context"> \
+                                   context</a> for the configuration.'))
```

Comment:
@R9295 please rewrite this as:

```python
context = JSONField(null=True,
                    blank=True,
                    help_text=_('Additional '
                    '<a href="http://netjsonconfig.openwisp.org/'
                    'en/stable/general/basics.html#context">'
                    'context</a> for the configuration.'))
```

when you split a string over multiple line you should do as indicated above otherwise it will result in what you see in the migration file, see my next comment